### ISSUE TITLE
Invalid DateTime from TMDB in call to Client.GetMovieAsync

### ISSUE THREAD

**Issue by SirSparkles** [LordMike/TMDbLib#390]
Whilst doing a Client.GetMovieAsync()

for movie 8967, 

Movie downloadedMovie = Client.GetMovieAsync(8967, "en", "en,null", MovieMethods.ExternalIds | MovieMethods.Images | MovieMethods.AlternativeTitles | MovieMethods.ReleaseDates | MovieMethods.Changes | MovieMethods.Videos | MovieMethods.Credits).Result;

Results in the following error:

FormatException: The string was not recognized as a valid DateTime. There is an unknown word starting at index 20.

Something changed for TMDB?

**Comment by SirSparkles** [comment]
Probably due to this: https://www.themoviedb.org/talk/616888d7d7f465005fa04bae ?

**Comment by SirSparkles** [comment]
> You you have the JSON response you received?

No, sorry, it's embedded in the library, so don't see the response directly. 

Looks like a fix is in place, but doesn't seem to have filtered out fully yet as the problem is still manifesting for me

**Comment by LordMike** [comment]
It should have filtered out by now, as I think Travis' cache only covers 8-24 hours. You can increase the log level to see the exact URL that is being hit, and then try that yourself to see the JSON body.

.. maybe I should introduce a "raw" dump of sorts to see the responses.. Maybe as a debug logging thing.
